### ISSUE TITLE
How to fine tune on this basis(wiki.zh.vec) in Python?

### ISSUE THREAD

**Issue by letotefrank** [facebookresearch/fastText#630]
I'm trying to fine tune on this basis(wiki.zh.vec) in Python, but the results is bad.The original word vector(wiki.zh.ve) became zero.

Note: 'my_data.txt' is my corpus.It is data sentence by sentence.
`# fit
model = train_unsupervised(  
   input=os.path.join('./data', 'my_data.txt'), 
    model='skipgram', 
    minCount=1,  
    dim=300, 
    pretrainedVectors="./vec/wiki.zh.vec"
)`

Thank you for you reply!

**Comment by EdouardGrave** [comment]
Hi @letotefrank,

Unfortunately, as of now, it is not possible to fine-tune pre-trained model with an unsupervised objective. The `pretrainedVectors` options should only be used with supervised models.

Best,
Edouard.
